stable/polaris/README.md

```diff
@@ -0,0 +1,29 @@
+# Polaris
+
+[Polaris](https://github.com/reactiveops/polaris)
+is a tool for auditing and enforcing best practices in Kubernetes.
+
+## Installation
+We recommend installing polaris in its own namespace.
+```
+helm install stable/polaris --name polaris --namespace polaris
```

Comment:
These instructions should look more like:

```
helm repo add reactiveops-stable https://charts.reactiveops.com/stable
helm install reactiveops-stable/polaris --name polaris --namespace polaris
```

Reply:
Hah, I was copying from RBAC manager

https://github.com/reactiveops/charts/pull/84

Reply:
Done